examples/aspect_based_sentiment/README.md

```diff
@@ -0,0 +1,22 @@
+# Aspect Based Sentiment Analysis
```

Comment:
I think it would be good to change the name of the folder to `sentiment_analysis`. This is more general and could add future contributions